File: surf-cloud-api/src/main/java/dev/slne/surf/cloud/api/netty/exception/SurfNettyListenerRegistrationException.java
package dev.slne.surf.cloud.api.netty.exception;

import java.io.Serial;
import lombok.experimental.StandardException;

@StandardException
public class SurfNettyListenerRegistrationException extends SurfNettyListenerException {

  @Serial
  private static final long serialVersionUID = 4077119187296696024L;
}

File: surf-cloud-api/src/main/kotlin/dev/slne/surf/cloud/api/netty/exception/SurfNettyListenerRegistrationException.kt
package dev.slne.surf.cloud.api.netty.exception

import java.io.Serial


class SurfNettyListenerRegistrationException : SurfNettyListenerException {
    constructor() : super()
    constructor(cause: Throwable?) : super(cause)
    constructor(message: String?) : super(message)
    constructor(message: String?, cause: Throwable?) : super(message, cause)

    companion object {
        @JvmStatic
        @Serial
        private val serialVersionUID = 4077119187296696024L
    }
}
